dataland-frontend/src/utils/Constants.ts

```diff
@@ -25,6 +25,13 @@ export const TIME_DELAY_BETWEEN_SUBMIT_AND_NEXT_ACTION_IN_MS = 2000;
 // This always has to be shorter than the ssoSessionIdleTimeout value in the realm settings.
 export const TIME_BEFORE_REFRESH_TOKEN_EXPIRY_TO_DISPLAY_SESSION_WARNING_IN_MS = 2 * 60 * 1000;
 
+/*
```

Comment:
What about a similar comment in the corresponding application.properties?

Reply:
Habe ein comment eingefügt, dass auf die Synchronität hinweist.